src/tests/JIT/HardwareIntrinsics/X86/Avx512F/LoadAlignedVector512NonTemporal.cs

```diff
@@ -0,0 +1,219 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
```

Comment:
Any reason for these empty `//` comment lines?

Reply:
Same setup as all the files currently have.